posthog/warehouse/api/saved_query.py

```diff
@@ -185,6 +212,8 @@ def safely_get_queryset(self, queryset):
     def destroy(self, request: request.Request, *args: Any, **kwargs: Any) -> response.Response:
         instance: DataWarehouseSavedQuery = self.get_object()
 
+        delete_saved_query_schedule(str(instance.id))
+
         for join in DataWarehouseJoin.objects.filter(
             Q(team_id=instance.team_id) & (Q(source_table_name=instance.name) | Q(joining_table_name=instance.name))
         ).exclude(deleted=True):
```

Comment:
There's an api defined below `def run`. I think if everything is going through the schedule, it would make sense to trigger the run through the schedule (for uniformity)

Reply:
It looks like the current setup is with non-scheduled syncs via the run endpoint and scheduled runs getting triggered/modified/deleted (since we now have the `never` option) by the update endpoint when the time changes as far as I can tell? I think that's reasonable enough?